hazelcast/src/test/java/com/hazelcast/test/annotation/IntegrationTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+/**
+ * Tests which should be executed after a final JAR is packaged.
+ * These tests are executed by the Fail-Safe Maven plugin.
+ * (other tests are executed by the Surefire plugin)
+ *
+ */
+public final class IntegrationTest {
```

Comment:
you are right. it's unnecessary now. 

I am actually thinking about removing the `IT`  suffix - as we don't use naming conventions for test categorization - we have `QuickTest`, `NightlyTest` and `SlowTest` assigned explicitly. Hence adding a new category - `IntegrationTest` - would fit into this pattern. (instead of introducing a new concept - categorization based on a name) 

WDYT?